src/Tests/AddThisFieldWebTestBase.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * @file
+ * Definition of Drupal\addthis\Tests\AddThisFieldWebTestBase.
+ *
+ * This code was taken from the field_example module in the
+ * D8 examples module. Would be awesome if something like this was provided
+ * by core.
+ */
+
+namespace Drupal\addthis\Tests;
+
+use Drupal\Core\Session\AccountInterface;
+use Drupal\simpletest\WebTestBase;
+
+class AddThisFieldWebTestBase extends WebTestBase {
```

Comment:
This is used as a helper class to setup the content type and add the AddThis field to it. This is needed in order to test the field settings/display settings for the field.